rocAL/rocAL/source/loaders/audio/node_audio_loader_single_shard.cpp

```diff
@@ -47,6 +47,10 @@ AudioLoaderSingleShardNode::init(unsigned shard_id, unsigned shard_count, unsign
     reader_cfg.set_batch_count(load_batch_count);
     reader_cfg.set_meta_data_reader(meta_data_reader);
     reader_cfg.set_cpu_num_threads(cpu_num_threads);
+    reader_cfg.set_last_batch_policy(last_batch_policy, last_batch_padded);
+    reader_cfg.set_stick_to_shard(stick_to_shard);
+    reader_cfg.set_shard_size(shard_size);
+    std::cerr << "\n In Node Audio Loader Single Shard :" << shard_size;
```

Comment:
Please remove the debugging print statement

Reply:
Removed